p22/tests/integration_tests.rs

```diff
@@ -0,0 +1,21 @@
+use p22::calc;
+
+#[test]
+fn test_temperature_conversions() {
+    assert_eq!(calc::celsius2farenheit(0), 32);
+    assert_eq!(calc::farenheit2celsius(32), 0);
+
+    // Test round trips
+    //let c = 37;
+    //assert_eq!(calc::farenheit2celsius(calc::celsius2farenheit(c)), c);
```

Comment:
Remove this? It's expected for the roundtrip to fail because of the rounding down.